content/nextflow-on-aws-batch/40_nextflow202/30_docker-image.md

```diff
@@ -0,0 +1,102 @@
+---
+title: "Nextflow Docker Image"
+chapter: false
+weight: 30
+---
+
+
+
+To check our setup we will run nextflow on the Cloud9 instance, submitting jobs to AWS Batch.
```

Comment:
Found this comment a bit misleading, is telling me about running nextflow on cloud 9 instance to submit jobs to AWS batch , but in this section the only thing I'm doing is (again) building a docker image that is the one that ultimately should be building up the image.

Reply:
changed the lead sentence.